server/routes/polls/update.route.js

```diff
@@ -0,0 +1,55 @@
+const express = require("express");
+const router = express.Router();
+const Poll = require("../../models/polls");
+const multer = require("multer");
+const aws = require("aws-sdk");
+const multerS3 = require("multer-s3");
+
+// Route to update the title of the poll
+router.post("/", (req, res) => {
+	const title = req.body.title;
+	const pollId = req.body.pollId;
```

Comment:
We should do some validation in these routes and add the JWT middleware. The validation should check if the user is the owner and they are allowed to update these polls. All of these routes should have this validation

Reply:
Yup. Agreed. I was waiting for the cookies backend to finish. I had already raised an issue in the icebox and assigned it to me to check whether a valid user is already logged in or not.